Others/MatlabFuzzy1.m

```diff
@@ -0,0 +1,263 @@
+clc

```

Comment:
Can you explain what this program is all about? Maybe add better comments and variable names?
If you are contributing more than one file then make sure to put them in one folder